.pre-commit-config.yaml

```diff
@@ -13,11 +13,15 @@ repos:
       - id: pretty-format-json
         exclude: ^$|.devcontainer
 
-  - repo: https://github.com/asottile/setup-cfg-fmt
-    rev: v2.2.0
+  - repo: https://github.com/tox-dev/pyproject-fmt
+    rev: "0.9.1"
     hooks:
-      - id: setup-cfg-fmt
+      - id: pyproject-fmt
 
+  - repo: https://github.com/abravalheri/validate-pyproject
+    rev: v0.12.1
+    hooks:
+      - id: validate-pyproject
```

Comment:
I would say to just stay with `pyproject-fmt`
```suggestion
```

Reply:
See the beta comments below.  Let's keep validating the `pyproject.toml` file given that its schema might still change